test/test_nvfuser_dynamo.py

```diff
@@ -0,0 +1,48 @@
+# Owner(s): ["module: nvfuser"]
```

Comment:
maybe put this in test/dynamo instead?

Reply:
Yes, I can change that. I just took a quick look at test/dynamo and there were no backend tests.